crates/sui-graphql-e2e-tests/tests/epoch/pagination.exp

```diff
@@ -0,0 +1,145 @@
+processed 17 tasks
+
+init:
+C: object(0,0)
+
+task 1, line 7:
+//# create-checkpoint
```

Comment:
nit: if the aim is just to create a checkpoint for the epoch to contain, that isn't necessary, `advance-epoch` also generates a checkpoint (as you can see, the checkpoints created by `create-checkpoint` seem to go up in 2's because of this).